packages/mdc-notched-outline/_mixins.scss

```diff
@@ -40,12 +41,12 @@
   }
 }
 
-@mixin mdc-notched-outline-corner-radius($radius) {
-  border-radius: $radius;
+@mixin mdc-notched-outline-shape-radius($radius, $rtl-reflexive: false) {
```

Comment:
We're not passing through `$rtl-reflexive` in these mixins

Reply:
Done.